proto/brpb.proto

```diff
@@ -876,7 +876,17 @@ message DeleteSpansOfFile {
     uint64 whole_file_length = 3;
 }
 
+enum MigrationVersion {
+    // The initial version. Somehow "UNKNOWN".
+    M0 = 0;
+    // Added `creator` and `version`.
+    M1 = 1;
+}
+
 message Migration {
+    string creator = 5;
```

Comment:
nit, maybe sort in the numeric order, put them to the end of the message.

Reply:
I think the `version` and `creator` field should be more primitive and essential than other fields. So I put them in the first line and people may notice them easier.